models/iaf_psc_alpha.cpp

```diff
@@ -340,9 +339,13 @@ iaf_psc_alpha::update( Time const& origin, const long from, const long to )
     S_.I_ex_ = V_.P21_ex_ * S_.dI_ex_ + V_.P22_ex_ * S_.I_ex_;
     S_.dI_ex_ *= V_.P11_ex_;
 
+    // get read access to the correct input-buffer slot
+    const index input_buffer_slot = kernel().event_delivery_manager.get_modulo( lag );
+    auto& input = B_.input_buffer_.get_values_all_channels( input_buffer_slot );
+
     // Apply spikes delivered in this step; spikes arriving at T+1 have
     // an immediate effect on the state of the neuron
-    V_.weighted_spikes_ex_ = B_.ex_spikes_.get_value( lag );
+    V_.weighted_spikes_ex_ = input[ Buffers_::SYN_EX ];
     S_.dI_ex_ += V_.EPSCInitialValue_ * V_.weighted_spikes_ex_;
```

Comment:
I think we recently decided to drop the `weighted_spikes_` recordables. Then, one can move the `input[ ... ]` right into line 349.

Reply:
Is there an issue documenting this decision? In any case, this is outside the scope of this PR.

Reply:
I fear it was only an informal, undocumented decision. I will create a separate issue for it.